rel/files/app.config

```diff
@@ -12,24 +16,21 @@
            {format_stacktrace_filter, {fun mongoose_log_filter:format_stacktrace_filter/2, no_state}}
         ]},
 
-    %% Console logger
-%   {handler, otp_shell_log, logger_std_h, #{
-%        formatter => {logger_formatter, #{
-%          depth => 12,
-%          chars_limit => 4096
-%     }}
-%   }},
+    %% Shell log handler:
     {handler, shell_log, logger_std_h, #{
+         %% Default log level for handlers is to log everything, that
+         %% passes primary log level and module log levels
+         level => all,
          formatter => {mongoose_flatlog_formatter, #{
            map_depth => 3,
-           term_depth => 50,
-           template_name => mongoose_default_log_template
+           term_depth => 50
          }}
     }},
 
-    %% Disk logger for errors
-    {handler, disk_log, logger_disk_log_h,
-       #{config => #{
+    %% File log handler:
+    {handler, disk_log, logger_disk_log_h, #{
+         level => all,
```

Comment:
It's better to see that default value clearly here, because system admins rarely read OTP Logger implementation docs.
I would easily assume that default value is the same as loglevel (nope).